plugins/www/Optimove.js

```diff
@@ -6,3 +6,29 @@ var exec = require('cordova/exec');
 window.initialize = function (inAppConsentStrategy, success, error) {
     exec(success, error, 'OptimoveSDKPlugin', 'initialize', [inAppConsentStrategy]);
 };
+window.setUserId = function (userId, success, error) {
```

Comment:
I think these should be exported from the module, rather than globally attached to the window.

The end user in the mobile app should really be consuming methods on the Optimove SDK, like `Optimove.setUserId(...)`.

Probably something like

```
module.exports = {
  setUserId: function setUseId() {},
  ...

};
```

For the public API, and then the `plugin.xml` sets the exported module name with the `clobbers` property on the JS module file.

Reply:
(see https://github.com/Kumulos/KumulosSdkCordova/blob/81a605e4f98afe103dd6ab1e9b03b75df1aa7007/plugin.xml)